### DISCUSSION TITLE
Change fonts in-line

### DISCUSSION THREAD

**Discussion by urbaman** [squidfunk/mkdocs-material#7144] [Q&A]
Hi,

I would really need a method to change the font just for a part of the text, like
`<font1>text in font1</font1><font2>text in font2</font2>`.

Is it doable? Should I use css? How to do it in markdown?

Thank you.

**Comment by alexvoss** [comment]
As far as I know, the only way would be to add HTML `span`tags with a suitable class. I had a look at the [pymdownx extensions](https://facelessuser.github.io/pymdown-extensions/) but could not find one that creates spans, though there are multiple extensions that produce more specific inline elements. If you can formulate a general use case, perhaps this is something you could suggest there? It is possible I am missing something, so perhaps see if anyone else comes up with something here.

**Reply by alexvoss** [reply]
P.S.: I would certainly be possible to write your own [hook](https://www.mkdocs.org/user-guide/configuration/#hooks) for this that scans the Markdown using a suitable regular expression and inserts the spans instead of whatever placeholders you chose.
